everclear.json

```diff
@@ -1060,6 +1060,55 @@
                     }
                 }
             }
+        },
+        "2020": {
+            "providers": [
+                "https://api.roninchain.com/rpc"
+            ],
+            "subgraphUrls": [
+                "https://api.goldsky.com/api/public/project_clssc64y57n5r010yeoly05up/subgraphs/everclear-spoke-ronin/production-v0.0.1/gn"
+            ],
+            "deployments": {
+                "everclear": "0xdCA40903E271Cc76AECd62dF8d6c19f3Ac873E64",
+                "gateway": "0x1FC1f47a6a7c61f53321643A14bEc044213AbF95",
+                "XERC20Module": "0x92dcaf947DB325ac023b105591d76315743883eD"
+            },
+            "confirmations": 1,
+            "assets": {
+                "RON": {
+                    "symbol": "RON",
+                    "address": "0x0000000000000000000000000000000000000000",
+                    "decimals": 18,
+                    "tickerHash": "0x3ed03c38e59dc60c7b69c2a4bf68f9214acd953252b5a90e8f5f59583e9bc3ae",
+                    "isNative": true,
+                    "price": {
+                        "isStable": false,
+                        "coingeckoId": "ron"
+                    }
+                },
+                "WETH": {
+                    "symbol": "WETH",
+                    "address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
```

Comment:
same concern for the weth address, vs https://app.roninchain.com/token/0xc99a6a985ed2cac1ef41640596c5a5f9f4e19ef5?p=1&ps=25